src/nbl/asset/utils/CHLSLCompiler.cpp

```diff
@@ -473,4 +567,108 @@ void CHLSLCompiler::insertIntoStart(std::string& code, std::ostringstream&& ins)
     code.insert(0u, ins.str());
 }
 
+
+
+
+// for including builtins 
+template<>
+inline bool boost::wave::impl::pp_iterator_functor<wave_context_t>::on_include_helper(char const* f, char const* s,
```

Comment:
I had to add a template specialization of this function to pretty much override it, as it would throw exceptions with builtins. Not the prettiest solution

Reply:
because you were checking for existence of paths, thats why if was throwing exceptions!


Remove this code!